optaplanner-core/src/main/java/org/optaplanner/core/impl/score/director/drools/testgen/CorruptedScoreException.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.optaplanner.core.impl.score.director.drools.testgen;
+
+public class CorruptedScoreException extends RuntimeException {
```

Comment:
Since the exception is currently only used internally by the TestGenCorruptedScoreReproducer I prefixed the exception with TestGen too.